packages/block-library/src/cover/edit.js

```diff
@@ -458,14 +538,26 @@ function CoverEdit( {
 				/>
 			</BlockControls>
 			<BlockControls group="other">
-				<MediaReplaceFlow
-					mediaId={ id }
-					mediaURL={ url }
-					allowedTypes={ ALLOWED_MEDIA_TYPES }
-					accept="image/*,video/*"
-					onSelect={ onSelectMedia }
-					name={ ! url ? __( 'Add Media' ) : __( 'Replace' ) }
-				/>
+				{ !! postId && (
+					<ToolbarButton
+						icon={ postFeaturedImage /*this is temporary*/ }
```

Comment:
Is it still temporary?

Reply:
Well ... IDK, it looks good enough for me.